doc/engineering-reference/src/simulation-models-encyclopedic-reference-002/demand-controlled-ventilation.tex

```diff
@@ -290,18 +290,31 @@ \subsection{Proportional Control}\label{proportional-control}
 {V_{ot - design,i}} = \frac{{\left( {{R_{p,i}}.{P_{z,i}} \times {SchedFrac_i}} \right) + ({R_{a,i}}.{A_{z,i}} \times {SchedFrac_i})}}{{{E_i}}}
 \end{equation}
 
+if ProportionalControlBasedOnDesignOARate is specified, the required intake flow rate will be revised as:
+
+ \begin{equation} {V{ot - design,i}} = \frac{Design Outdoor Air Flow Rate_i} {E_i} \end{equation}
```

Comment:
Your subscripts aren't showing up as subscripts here.

```
{V{ot - design,i}}
```

needs to be

```
{V_{ot - design,i}}
```

If that's the only real issue I'll make that change since I'm already working on it for the transition.